dependencies.props

```diff
@@ -17,15 +17,18 @@
   </PropertyGroup>
 
   <PropertyGroup>
+    <MicrosoftNETCorePlatformsPackageVersion>2.1.0-preview1-25824-02</MicrosoftNETCorePlatformsPackageVersion>
     <MicrosoftPrivateCoreFxNETCoreAppPackageVersion>4.5.0-preview1-25824-02</MicrosoftPrivateCoreFxNETCoreAppPackageVersion>
     <MicrosoftPrivateCoreFxUAPPackageVersion>4.6.0-preview1-25824-02</MicrosoftPrivateCoreFxUAPPackageVersion>
-    <PlatformPackageVersion>2.1.0-preview1-25824-02</PlatformPackageVersion>
     <MicrosoftNETCoreRuntimeCoreCLRPackageVersion>2.1.0-preview1-25824-51</MicrosoftNETCoreRuntimeCoreCLRPackageVersion>
-    <MicrosoftNETCoreRuntimeJitPackageVersion>$(MicrosoftNETCoreRuntimeCoreCLRPackageVersion)</MicrosoftNETCoreRuntimeJitPackageVersion>
-    <MicrosoftNetNativeCompilerPackageVersion>2.0.0-preview-25524-00</MicrosoftNetNativeCompilerPackageVersion>
-    <NETStandardVersion>2.1.0-preview1-25825-02</NETStandardVersion>
-    <DiaSymReaderNativeVersion>1.4.1</DiaSymReaderNativeVersion>
-    <WcfVersion>4.5.0-preview2-25825-02</WcfVersion>
+    <NETStandardVersion>2.1.0-preview1-25824-08</NETStandardVersion>
+    <MicrosoftDiaSymReaderNativePackageVersion>1.4.1</MicrosoftDiaSymReaderNativePackageVersion>
+    <WcfVersion>4.5.0-preview2-25824-01</WcfVersion>
```

Comment:
This `WcfVersion` property should be removed: the updater below changed to `SystemServiceModelDuplexPackageVersion`. (Probably auto-updated on `master` and caused a merge conflict.)